crates/uv-cli/src/lib.rs

```diff
@@ -3351,6 +3351,10 @@ pub struct ToolInstallArgs {
     #[arg(long)]
     pub with: Vec<String>,
 
+    /// Additionally include command entrypoints from the following packages.
+    #[arg(long)]
+    pub with_commands_from: Vec<PackageName>,
```

Comment:
I noticed you're following "install commands" from the other docstrings but we say "executables" elsehwere in the documentation and I think I prefer that i.e.

```suggestion
    /// Install executables from an additional package.
    #[arg(long)]
    pub with_executables_from: Vec<PackageName>,
```

I can change the other option documentation to match.